timemachine/ff/__init__.py

```diff
@@ -33,11 +33,11 @@ def combine_params(a: ForcefieldParams[_T], b: ForcefieldParams[_T]) -> Forcefie
         (a.pt_params, b.pt_params),
         (a.it_params, b.it_params),
         (a.q_params, b.q_params),
-        (a.q_params_solv, b.q_params_solv),
         (a.q_params_intra, b.q_params_intra),
+        (a.q_params_solv, b.q_params_solv),
         (a.lj_params, b.lj_params),
-        (a.lj_params_solv, b.lj_params_solv),
         (a.lj_params_intra, b.lj_params_intra),
+        (a.lj_params_solv, b.lj_params_solv),
     )
```

Comment:
Don't immediately have any ideas for making this safer with types that don't involve unwieldy type annotations. One way to catch this would be to give each field in `ForcefieldParams` a distinct type (e.g. `HarmonicBondParams = NewType("HarmonicBondParams", NDArray)`), but this would make `combine_params` very awkward to type.